provider/resource_harbor_project.go

```diff
@@ -114,8 +123,29 @@ func resourceProjectUpdate(d *schema.ResourceData, meta interface{}) error {
 
 func resourceProjectDelete(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*harbor.Client)
+	projectName := d.Get("name").(string)
+
+	repos, err := client.GetRepositories(d.Get("projectid").(string))
+	if err != nil {
+		return err
+	}
+
+	err = client.DeleteRepositories(repos)
```

Comment:
can we add a check to see if `repos` is empty before making an API call that might be unnecessary?

Reply:
Done